### ISSUE TITLE
Items Array is Empty When Only A Single Item Exists

### ISSUE THREAD

**Issue by ryanrossiter** [nasa8x/rss-to-json#25]
**Problem:** The returned `items` property is an empty array when fetching from sources that only have a single item present. (Did something change recently? This was working for me previously)

**Expected Behaviour:** For sources with one item, the items property of the returned object should be an array with a single element that is the parsed item.

**Reproduction:**
```javascript
const Feed = require('rss-to-json');

// this RSS link may have more than one item in the future, ATOW it only has one
Feed.load('https://anchor.fm/s/30e91ea4/podcast/rss')
.then((rssData) => {
  if (rssData.items.length === 0) console.error('Items is empty!');
});
```

**Investigation:** When an RSS feed returns a single `<item>` entry, the [`channel.item`](https://github.com/nasa8x/rss-to-json/blob/master/src/rss.js#L56) value is the object, not an array as it would be when multiple `<item>` entries are present.

**Solution:** Add a type check on the `items` value to check if it's an array, and coerce it to an array if it is not.

**Comment by ryanrossiter** [comment]
Oops - just looked at the `parser` function on the master branch and I see that this problem has been resolved.

Closing this issue, thanks!

**Comment by chapuyj** [comment]
It is on master, but not published on npm (1.1.1).
